BetaCameras/WebCam/WebCam.vcxproj

```diff
@@ -181,6 +182,9 @@
   </ItemGroup>
   <ItemGroup>
     <Image Include="app.ico" />
+    <Image Include="WebcamIcon.ico">
+      <DeploymentContent>true</DeploymentContent>
```

Comment:
I assume that's the C++ equivalent to CopyToOutputDirectory and can (if the assumption is true) be removed.